apps/common/src/main/scala/org/lfdecentralizedtrust/splice/automation/SqlIndexInitializationTrigger.scala

```diff
@@ -0,0 +1,316 @@
+    "dso_txlog_store_sid_et_an_e" -> IndexAction.ManagedByFlyway,
+    "active_parties_pkey" -> IndexAction.ManagedByFlyway,
+    "active_parties_sid_p_cr_idx" -> IndexAction.ManagedByFlyway,
+    "update_history_unassignments_pkey" -> IndexAction.ManagedByFlyway,
+    "updt_hist_unas_hi_mi_di_rt" -> IndexAction.ManagedByFlyway,
+    "updt_hist_unas_hi_mi_rt_di" -> IndexAction.ManagedByFlyway,
+    "updt_hist_unas_hi_mi_ui_u" -> IndexAction.ManagedByFlyway,
+    "update_history_assignments_pkey" -> IndexAction.ManagedByFlyway,
+    "updt_hist_assi_hi_mi_di_rt" -> IndexAction.ManagedByFlyway,
+    "updt_hist_assi_hi_mi_rt_di" -> IndexAction.ManagedByFlyway,
+    "updt_hist_assi_hi_mi_ui_u" -> IndexAction.ManagedByFlyway,
+    "update_history_backfilling_pkey" -> IndexAction.ManagedByFlyway,
+    "round_party_totals_pkey" -> IndexAction.ManagedByFlyway,
+    "acs_snapshot_data_pkey" -> IndexAction.ManagedByFlyway,
+    "acs_snapshot_data_all_filters" -> IndexAction.ManagedByFlyway,
+    "acs_snapshot_data_template_only_filter" -> IndexAction.ManagedByFlyway,
+    "acs_snapshot_data_stakeholder_only_filter" -> IndexAction.ManagedByFlyway,
+    "acs_snapshot_data_cid" -> IndexAction.ManagedByFlyway,
+    "acs_snapshot_pkey" -> IndexAction.ManagedByFlyway,
+    "external_party_wallet_acs_store_pkey" -> IndexAction.ManagedByFlyway,
+    "external_party_wallet_acs_sto_store_id_migration_id_contrac_idx" -> IndexAction.ManagedByFlyway,
+    "external_party_wallet_acs_sto_store_id_migration_id_templat_idx" -> IndexAction.ManagedByFlyway,
+    "external_party_wallet_acs_sto_store_id_migration_id_templa_idx1" -> IndexAction.ManagedByFlyway,
+    "external_party_wallet_acs_sto_store_id_migration_id_state_n_idx" -> IndexAction.ManagedByFlyway,
+    "external_party_wallet_acs_sto_store_id_migration_id_templa_idx2" -> IndexAction.ManagedByFlyway,
+    "external_party_wallet_acs_store_sid_mid_tid_rcr" -> IndexAction.ManagedByFlyway,
+    "scan_txlog_store_sid_et_ei" -> IndexAction.ManagedByFlyway,
+    "scan_txlog_store_pkey" -> IndexAction.ManagedByFlyway,
+    "scan_txlog_store_store_id_migration_id_domain_id_record_tim_idx" -> IndexAction.ManagedByFlyway,
+    "scan_txlog_store_sid_irt_r_en" -> IndexAction.ManagedByFlyway,
+    "scan_txlog_store_sid_mid_did_rt" -> IndexAction.ManagedByFlyway,
+    "scan_txlog_store_sid_tfer_cmd_sender_nonce" -> IndexAction.ManagedByFlyway,
+  )
+
+  /** Indexes managed by this trigger class */
+  val customIndexes: Map[String, IndexAction] = Map(
+    "updt_hist_crea_hi_mi_ci_import_updates" -> IndexAction.Create(sqlu"""
+        create index concurrently if not exists updt_hist_crea_hi_mi_ci_import_updates
+        on update_history_creates (history_id, migration_id, contract_id)
+        where record_time = -62135596800000000
+         """)
+  )
+
+  val defaultExpectedIndexes: Map[String, IndexAction] = flywayIndexes ++ customIndexes
```

Comment:
how do you verify that the custom index is already covered by the flyway indexes (index exists on the same columns)